packages/core/src/components/ImageViewer/index.tsx

```diff
@@ -158,6 +154,11 @@ export default withStyles(({ ui }) => ({
   },
 
   image: {
-    display: 'inline-block',
+    width: '100%',
+    height: '100%',
+    display: 'flex',
+    align: 'center',
```

Comment:
`align` isn't necessary or valid for `flex`. i see that it was there before - sorry we didn't catch it the first time around!